buffer/src/test/java/io/netty/buffer/AbstractPooledByteBufTest.java

```diff
@@ -49,11 +49,10 @@ private void testDiscardMarks(int capacity) {
 
         buf.markReaderIndex();
         buf.markWriterIndex();
-        assertTrue(buf.release());
```

Comment:
what do we want to do with this test? it seems like `buf` is not really used to assert anything here?